vtr_flow/scripts/python_libs/verilogtorouting/flow.py

```diff
@@ -20,7 +21,9 @@ def run_vtr_flow(architecture_file, circuit_file,
                  parse_config_file=None,
                  temp_dir="./temp", 
                  verbosity=0,
-                 vpr_args=None):
+                 vpr_args=None,
+                 flow_type=2,
```

Comment:
Can we give this a name instead of a number? I'm not sure what flow_type = 2 means, but a name could be much easier to understand.

Reply:
This has been fixed